tasks/task105_story_completion.json

```diff
@@ -1,7 +1,7 @@
 {
     "Contributors": "Mirali Purohit",
     "Categories": "Sentence Generation",
-    "Definition": "In this task, you're given a story which is incomplete. Each story has four sentences written in natural language. Your job is to complete the end part of an incomplete story by predicting appropriate last sentence.",
+    "Definition": "In this task, you're given a four sentences of story written in natural language. Your job is to complete end part of the story by predicting appropriate last sentence.",
```

Comment:
add something along this line: .... in a manner that is coherent with the provided sentences (?)

Reply:
Please review again.